### PR TITLE
[_] feature/add loader while switching workspace

### PR DESCRIPTION
**Pull request by rafijv**
- display spinner while switching workspace

### CODE REVIEW

**Review by CandelR — CHANGES_REQUESTED** [review]
@rafijv  I have asked for a design to make this look better and have a better UX

File: src/app/store/slices/workspaces/workspacesStore.ts [hunk 1]
```diff
@@ -97,6 +100,7 @@ const fetchCredentials = createAsyncThunk<void, undefined, { state: RootState }>
       dispatch(workspacesActions.setCredentials(credentials));
       localStorageService.set(STORAGE_KEYS.WORKSPACE_CREDENTIALS, JSON.stringify(credentials));
     }
+    dispatch(workspacesActions.setIsLoadingCredentials(false));
```
Comment (CandelR) [inline src/app/store/slices/workspaces/workspacesStore.ts:103]: we have to manage when this fails so that the loader doesn't get stuck in infinity.

Comment (rafijv) [reply src/app/store/slices/workspaces/workspacesStore.ts:103]: Done

**Comment by rafijv** [comment]
> @rafijv I have asked for a design to make this look better and have a better UX

Done

**Review by CandelR — APPROVED** [review]

**Pull request merged**